rpc/rpcserver/server.go

```diff
@@ -2750,3 +2750,27 @@ func (s *decodeMessageServer) DecodeRawTransaction(ctx context.Context, req *pb.
 
 	return resp, nil
 }
+
+func (s *walletServer) BestBlock(ctx context.Context, req *pb.BestBlockRequest) (
+	*pb.BestBlockResponse, error) {
+	n, err := s.requireNetworkBackend()
```

Comment:
We should be querying the wallet's best block, not the network's.  This does not require access to the network backend.  Use https://godoc.org/github.com/decred/dcrwallet/wallet#Wallet.MainChainTip for this